copt/frank_wolfe.py

```diff
@@ -220,23 +226,23 @@ def minimize_frank_wolfe(
                 update_direction,
                 norm_update_direction,
             )
-        elif step_size == "DR":
+        elif step == "DR":
```

Comment:
DR? Seems to be minimizing the quadratic upper bound when L is known.

Reply:
Demyanov-Rubinov, default when step size is unspecified but lipschitz is.

Reply:
Thanks for the suggestion, I updated the description. On the second comment, not sure what you mean by unspecified. I think it will only use this step-size if ```step='DR'```